index.html

```diff
@@ -31,14 +31,23 @@
   </nav>
   <main class= web-content>
     <section class = content-container>
```

Comment:
Make sure you don't have spaces here. It should look like this:
    <section class=content-container>

Reply:
Thanks for the feedback. I have fixed these but still my :
<section>
        <ul>
          <li><h1 class="karma-title">Everyone needs a little Karma</h1></li>
          <li><img class='device' src ='img/icon-devices.svg'></li>
          <li></li>
          <li></li>
          <li></li>
          <li></li>
        </ul>
     
    </section>
is hidden under the main background.

Reply:
Your classes all need quotation marks around them @Alisina92 
`<main class="web-content">`
`<section class="content-container">`

You have typing errors on most lines. As Chris said:
- You should never have spaces when you are writing HTML attributes (for example adding class, or src, or href). The format should always be attributename="attributevalue"
e.g. `class="main-content"`
- You need the quotation marks